src/types/project.rs

```diff
@@ -67,6 +67,7 @@ pub struct ProjectDetailsResponse {
 #[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Debug, Serialize, Deserialize)]
 pub struct CreateProjectRequest {
     pub name: String,
+    pub group_name: Option<String>
```

Comment:
The new optional field for future create project requests to use as the project owner. API service will do the lookup to translate the `group_name` to the underlying uuid.